### PR TITLE
Update mjml to 3.3.2.

### PR DESCRIPTION
**Pull request by qkevinto**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
 const mjml = require('mjml');
+const colors = require('colors');
```
Comment (ramasilveyra) [inline index.js:2]: This require isn't used, could you remove it please?

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/ramasilveyra/mjml-loader#readme",
   "dependencies": {
-    "mjml": "^2.3.0"
+    "colors": "^1.1.2",
```
Comment (ramasilveyra) [inline package.json:25]: Also could you remove this dep from the package.json please?

File: index.js [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
 
-module.exports = content => {
-  this.cacheable && this.cacheable();
-  this.value = content;
-  return `module.exports = ${JSON.stringify(mjml.mjml2html(content))};`;
+module.exports = function(content) {
```
Comment (ramasilveyra) [inline index.js]: Good catch! One little comment, if you want could you please add a name to the anonymous function please?

```js
module.exports =  function mjmlLoader(content) {
```

So debugging tools work better with mjml-loader

File: index.js [hunk 4]
```diff
@@ -1,7 +1,20 @@
...
+  this.cacheable();
+
+  const result = mjml.mjml2html(content);
+
+  if (result.errors.length) {
+    console.log(`[mjml-loader] ERROR in ${this.resourcePath}:`.red);
```
Comment (ramasilveyra) [inline index.js]: Wow I didn't know about the `.red` on the `console.log`!! Really good!

**Review by ramasilveyra — COMMENTED** [review]
Thanks! I'm eager to merge this, there is a lot of improvements!

I left some comments, also I see that your commits aren't associated to your GitHub account if this is unintentional you can fix this with https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/#you-havent-added-your-local-git-commit-email-to-your-profile + recreating the commits or http://stackoverflow.com/a/3042512/4709891.

And if you don't have time to finish this let me know, I can make time to finish it.

Thanks again!

File: index.js (hunk 1 above)
Comment (qkevinto) [reply index.js:2]: I'm using this package to colourize the outputted error. The documentation for the [color](https://www.npmjs.com/package/colors) package notes that it extends `String.prototype` hence why I could use `.red` method on a string on [Line 10](https://github.com/ramasilveyra/mjml-loader/pull/1/files/980c1381b4ae14b5d28159921be3ebf0da115aaf#diff-168726dbe96b3ce427e7fedce31bb0bcR10) 😆. But I'll make it more explicit by using the package function and using the colour's method instead.

File: index.js (hunk 4 above)
Comment (qkevinto) [reply index.js]: See my other comment above hehe.

**Comment by qkevinto** [comment]
Ahh, thanks for the note on commits not being associated with my account. I was using a different machine. 🙂

I believe I have addressed all your comments, thanks for the review! Let me know if there's anything else.

File: package.json (hunk 2 above)
Comment (qkevinto) [reply package.json:25]: Please see my comment above regarding this package.

File: index.js (hunk 1 above)
Comment (ramasilveyra) [reply index.js:2]: It's true, my bad! Explicit it's better

**Comment by ramasilveyra** [comment]
Thanks for your time on this! Released under `1.0.0-beta.1`

**Pull request merged**